common/flatpak-dir.c

```diff
@@ -4815,18 +5008,14 @@ flatpak_dir_run_triggers (FlatpakDir   *self,
   if (flatpak_dir_use_system_helper (self, NULL))
     {
       const char *installation = flatpak_dir_get_id (self);
-      FlatpakSystemHelper *system_helper = flatpak_dir_get_system_helper (self);
 
       /* If we don't have the system helper, we'll have to try as an
        * unprivileged user, which might fail later */
```

Comment:
I wonder if this comment means we should not error out if the system helper isn't available. Otherwise it should be removed if it's no longer relevant.

It doesn't really make sense though, because it seems not to try as an unprivileged user if `flatpak_dir_use_system_helper` returned TRUE.

Reply:
Yeah, i think these should all be removed